test/ci/kokoro/windows/continuous.cfg

```diff
@@ -0,0 +1,42 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+build_file: "test/ci/kokoro/windows/run_integ_tests.ps1"
+timeout_mins: 40
```

Comment:
Does this timeout include VM setup time?  I ask because the tests take ~25 min to run on my Windows machine, and that's only including from the time I press enter to the time they finish... and that's when none of the requests had to be retried, etc.

How do you feel about bumping this to 50 or 60? That's 2x what we've observed it normally takes in the average case, and should account for any setup hangups or 10-minute retry loops.

Reply:
Good point on startup time! I'm not sure how long exactly this will take until we get some successful builds. I'll bump it to 60 across the board and we can adjust as we gather some data.

Reply:
Ok, pushed a commit bumping them all to 60m